Space-Invaders-Js/README.md

```diff
@@ -0,0 +1,3 @@
+# Space-Invaders
```

Comment:
- Add Gif in `readme.md` 
- Add instruction in `readme.md`
- Add author's name in `readme.md`

Reply:
hey there,
i added the requested changes.